crates/scroll/node/src/addons.rs

```diff
@@ -118,23 +127,94 @@ where
 }
 
 /// A regular scroll evm and executor builder.
-#[derive(Debug, Default, Clone)]
+#[derive(Debug, Clone)]
 #[non_exhaustive]
-pub struct ScrollAddOnsBuilder {}
+pub struct ScrollAddOnsBuilder<NetworkT, RpcMiddleware = Identity> {
+    /// Sequencer client, configured to forward submitted transactions to sequencer of given Scroll
+    /// network.
+    sequencer_url: Option<String>,
+    /// Minimum suggested priority fee (tip)
+    min_suggested_priority_fee: u64,
+    /// Maximum payload size
+    payload_size_limit: u64,
+    /// Marker for network types.
+    _nt: PhantomData<NetworkT>,
+    /// RPC middleware to use
+    rpc_middleware: RpcMiddleware,
+}
+
+impl<NetworkT> Default for ScrollAddOnsBuilder<NetworkT> {
+    fn default() -> Self {
+        Self {
+            sequencer_url: None,
+            payload_size_limit: SCROLL_DEFAULT_PAYLOAD_SIZE_LIMIT,
+            // TODO (scroll): update with default values.
+            min_suggested_priority_fee: 1_000_000,
```

Comment:
Do we plan to change this before merging or after? If the later then lets create an issue to track this.

Reply:
I'm waiting for PR #291 to be merged, these changes were added just to make the later merge easier.